src/sentry/static/sentry/app/components/sidebar/index.jsx

```diff
@@ -300,6 +300,16 @@ class Sidebar extends React.Component {
                   to={`/organizations/${organization.slug}/stats/`}
                 />
               </SidebarSection>
+
+              <SidebarSection>
```

Comment:
Should this be feature flagged with `sentry10`? FYI there's one in the sidebar with the `sentry10` flag already.

Reply:
I'm leaning towards no flag. there's not really a new feature here, just a slight usablity boost from surfacing the settings icon and making the layout more consistent with the rest of the app

Reply:
Let's remove the SidebarItem with the sentry10 flag then

Reply:
Ah, I didn't realize you meant there was already _Settings_ icon with the flag. TIL. I'll remove it.